Content.Server/Light/EntitySystems/ExpendableLightSystem.cs

```diff
@@ -134,8 +135,8 @@ private void UpdateVisualizer(Entity<ExpendableLightComponent> ent, AppearanceCo
 
                 case ExpendableLightState.Dead:
                     _appearance.SetData(ent, ExpendableLightVisuals.Behavior, string.Empty, appearance);
-                    var isHotEvent = new IsHotEvent() {IsHot = true};
-                    RaiseLocalEvent(ent, isHotEvent);
+                    var ignite = new IgnitionEvent(true);
```

Comment:
Can you test to see if this should be true or false?

Does it still light things when the flare runs out?

Reply:
It does still light things when the flare goes out. I uh... don't think they should though, it does not feel consistent with other burnt out items like burnt matches and cigarette ends not lighting stuff. Flares are hotter but there probably should not be a common, undetectable, fire-starting item that can be shoved into a plant pot and left forever to set any passing flammable gas on fire.

Reply:
Probably a mistake then. I'd make it false.